crates/dojo-physics/examples/projectile/cairo_project.toml

```diff
@@ -0,0 +1,3 @@
+[crate_roots]
```

Comment:
can we remove this?

Reply:
Yes. Remove the entire file? Just lmk and i'll take care of it

Reply:
Good point about making some of these into common library functions. This could work for these functions but I'm not sure if we can use generics:

- `fn deg_to_rad` (uses Fixed type constants)
- `fn vec2_from_mag_theta` (uses functions from Cubit for Fixed type)
- `fn calc_x` and `calc_y` could be merged into a single library function `calc_position` (would use Fixed type constant `half`)

Lmk what you think

Reply:
Yes we can remove the entire file. Pulling out the methods makes sense. Perhaps it is better to stick to Fixed rather than using generics for now. Is there any logic from cubit we are duplicating here?

Reply:
I think some of these can go in vec2 in cubit, and I can make it point there. I don't think these functions duplicate anything currently in cubit.

should I leave the vec2.cairo currently in dojo-physics, in case we want to go to generics later?